packages/api/tests/endpoints/test-replays.js

```diff
@@ -170,3 +195,75 @@ test.serial('request to /replays endpoint returns 503 if starting ECS task throw
     asyncOperationStartStub.restore();
   }
 });
+
+test.serial('POST /replays/sqs starts an async-operation with specified payload', async (t) => {
+  const { queues } = t.context;
+  const asyncOperationStartStub = sinon.stub(asyncOperations, 'startAsyncOperation').returns(
+    new Promise((resolve) => resolve({ id: randomId('asyncOperationId') }))
+  );
+  asyncOperationStartStub.resetHistory();
```

Comment:
We don't need to call `resetHistory` in each test if each stub is specific to that test, which seems to be the case for all these tests

Reply:
Oh, okay yeah that makes sense [`030ed97` (#2373)](https://github.com/nasa/cumulus/pull/2373/commits/030ed97f35a8892454a2f46e229a87b2bb447203#diff-df2af8d19c0a84c9334f9bf37ffe602f1e7af349aee3c218576a9d3c6cb4590eL239)